proposals/0091-SDLScreen-SDLWindow-Projection.md

```diff
@@ -43,68 +43,33 @@ The SDLCarWindow interface is defined as follows:
 
-```objc
-extern NSString *const SDLVideoStreamDidStartNotification;
-extern NSString *const SDLVideoStreamDidStopNotification;
-
-extern NSString *const SDLAudioStreamDidStartNotification;
-extern NSString *const SDLAudioStreamDidStopNotification;
-
-@class SDLManager;
-
-@interface SDLStreamingMediaLifecycleManager : NSObject
-
-@property (nonatomic, weak, nullable) SDLManager *sdlManager;
-
-@property (nonatomic, assign, readonly, getter=isVideoConnected) BOOL videoConnected;
-@property (nonatomic, assign, readonly, getter=isAudioConnected) BOOL audioConnected;
-
-@property (nonatomic, assign, readonly, getter=isVideoStreamingAllowed) BOOL videoStreamingAllowed;
-
-@end
-```
+The SDLCarWindow class will also include the previously app-level common logic that interacts with the existing SDLStreamingMediaManager class to stream pixel buffers and receive audio data. It relies on the helper class SDLStreamingMediaLifecycleManager to ensure streaming is allowed and manage the streaming state.
 
 ## Potential downsides
 
-In choosing a projection API that differs from the standard UIScreen/UIWindow model for displaying content on an external screen, we are asking developers to adopt a model that is somewhat unfamiliar. However, it is also similar enough that the learning curve should be short and it avoids the possibility of strange side-effects due to subclassing UIKit controls.
+In choosing a projection API that differs from the standard UIScreen/UIWindow model for displaying content on an external screen, we are asking developers to adopt a model that is somewhat unfamiliar. 
 
 ## Impact on existing code
 
-This proposal has no impact on existing video/audio streaming applications. It provides a new higher level interface that should improve developer productivity and a foundation for both touch and physical button focus/select interactions.
+This proposal has no impact on existing video/audio streaming applications. It provides a new higher level interface that should improve developer productivity and a foundation for video projection.
 
 ## Alternatives considered
 
-This proposal provides no support for SDLCarWindow and SDLInterfaceManager encoding of pure OpenGL interfaces. Since all OpenGL content must be displayed within a UIView on iOS, it might be possible to intelligently encode video for the OpenGL layer(s) in SDLCarWindow. The SDLInterfaceManager could expose a CGRect collection property for defining the haptic spatial regions of an OpenGL UI.
+This proposal provides no support for SDLCarWindow encoding of pure OpenGL interfaces. Since all OpenGL content must be displayed within a UIView on iOS, it might be possible to intelligently encode video for the OpenGL layer(s) in SDLCarWindow. \
```

Comment:
This is very minor, but can you please remove the extra \ ?

Reply:
Removed